lib/api/core/abstractManifest.js

```diff
@@ -46,7 +46,11 @@ class AbstractManifest {
     try {
       this.raw = require(this.manifestPath);
     } catch (e) {
-      throw new PluginImplementationError(`[${this.path}] Unable to load the file 'manifest.json'`);
+      errorsManager.throw(
+        'plugins',
+        'runtime',
```

Comment:
Checks on the manifest.json file should be in the `validation` subdomain (this means that error and the one below).